src/gallium/state_trackers/nine/device9.c

```diff
@@ -57,20 +57,38 @@
 
 #if defined(PIPE_CC_GCC) && (defined(PIPE_ARCH_X86) || defined(PIPE_ARCH_X86_64))
 
-#include <fpu_control.h>
+#include <features.h>
+
+typedef unsigned int fpu_control_t __attribute__ ((__mode__ (__HI__)));
+
```

Comment:
GNU extension.(Clang supported)
#include is for __mode__(__HI__)
It means an integer twice as wide as the smallest addressable unit, usually 16 bits(sizeof(char) * 2).
Original code is defined this way, i guess they want it not to rely on libc and cross-platform.So i kept it.

On x86,we can use "unsigned short" or "uint16_t" to replace it.
this is a test program, the same result with -m32
```
char 1
uint 4
uint16_t 2
ushort 2
fpu_control_t 2
```